src/main/java/dk/bierproductie/opc_ua_client/enums/Commands.java

```diff
@@ -0,0 +1,16 @@
+package dk.bierproductie.opc_ua_client.enums;
+
+public enum Commands {
+    ZERO("Zero"),
+    RESET("Reset"),
+    START("Start"),
+    STOP("Stop"),
+    ABORT("Abort"),
+    CLEAR("Clear");
+
+    public final String output;
+
+    Commands(String output) {
+        this.output = output;
```

Comment:
Why would you this.output = output? 
not sure what it will be used for? 
I don't think it is a fault just curies

Reply:
it's a string that can be used as output for the enum values. So when you want the enum value Prouducts[0] it can print "Pilsner" instead of PILSNER